src/gameplay_v2/gameTypes.ts

```diff
@@ -7,3 +7,18 @@ export class GameMoveData {
   type: string;
   data: any;
 }
+
+export enum Role {
+  Resistance,
+  Spy,
+  Merlin,
+  //add more if you wish to
+}
+
+export enum Round {
+  StartingRound,
+  Round_2,
+  Round_3,
+  Round_4,
+  FinalRound,
```

Comment:
These are Missions. Do you need to define this as an enum? We don't have this currently, but we *might* have 6 missions by some random effect so not sure if we need to enum this unless we have a good reason to?

Reply:
Aaah, I was thinking like since there are cards like lady of the lake which was executed in Round 2, so I was thinking to put that in an enum. But, I'll try another way.